src/command.ts

```diff
@@ -94,9 +94,13 @@ export default class CommandHelp {
     if (!args.filter(a => a.description).length) return
     let body = renderList(args.map(a => {
       const name = a.name.toUpperCase()
+      let options
+      if (a.options) {
+        options = `Can be one of: ${a.options.join(', ')}`
+      }
       let description = a.description || ''
       if (a.default) description = `[default: ${a.default}] ${description}`
```

Comment:
can you instead do:

```ts
if (a.options) description = `(${a.options.join('|')}) ${description}`
```

This will better match how we show flag options.

Also, passing in a 3-length array to the list is likely not going to work in all cases. That behavior I plan to remove as we're not actually using it anyways I don't think.

Reply:
Updated.